test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1706,14 +1620,13 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
 
     # Validate all available user agents for Invoke-RestMethod
     $agents = @{InternetExplorer = "MSIE 9.0"
```

Comment:
Maybe you want to move this hash table entry to a new line, like the other `$agents` hashtable.

Reply:
Good catch. Thanks! Fixed.